templates/user/overview.html.twig

```diff
@@ -11,7 +11,7 @@
 {% endblock %}
 
 {% block sidebar_top %}
-    {% if (is_granted('ROLE_ADMIN') or is_granted('ROLE_MODERATOR')) and app.user != user %}
+    {% if (is_granted('ROLE_ADMIN') or is_granted('ROLE_MODERATOR')) and app.user != user and 'ROLE_ADMIN' not in user.roles %}
```

Comment:
it seems I'm the first one to use `in user.roles` which really surprised me (and also sets off red flags). I'm not sure if there is any reason it was avoided

Reply:
seems this was accurate, @BentiGorlich found an easier / perhaps more accurate if ever changes way of calling `.admin()` which calls `isAdmin()` (which basically does the same thing `return \in_array('ROLE_ADMIN', $this->getRoles());`) but seems safer way of doing it